deps/Makefile

```diff
@@ -728,6 +728,8 @@ $(eval $(call LLVM_PATCH,llvm-3.8.0_winshlib))
 $(eval $(call LLVM_PATCH,llvm-3.8.0_threads))
 # fix replutil test on unix
 $(eval $(call LLVM_PATCH,llvm-D17165-D18583))
+# Segfault for aggregate load
+$(eval $(call LLVM_PATCH,llvm-D17326_unpack_load))
```

Comment:
touches the same file as `llvm-D14260.patch`, should have a dependency to avoid a race condition